.github/workflows/check-fmt.yml

```diff
@@ -0,0 +1,34 @@
+name: check fmt
+
+on:
+  pull_request:
+      types: [opened, synchronize, reopened]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+jobs:
+  check-fmt:
+    runs-on: ubuntu-latest
+    container:
+      image: paritytech/ci-unified:bullseye-1.77.0
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up caching for Cargo
```

Comment:
I think it's an overkill to use a cache for `cargo fmt`. Let's remove it from here.

Reply:
removed